lxd/device/device_utils_unix_events.go

```diff
@@ -140,18 +140,7 @@ func unixRunHandlers(state *state.State, event *UnixEvent) {
 }
 
 // unixNewEvent returns a newly created Unix device event struct.
-// If an empty action is supplied then the action of the event is derived from whether the path
-// exists (add) or not (removed). This allows the peculiarities of the inotify API to be somewhat
-// masked by the consuming event handler functions.
-func unixNewEvent(action string, path string) UnixEvent {
```

Comment:
did you test this with inotify and not just fanotify?

Reply:
That's a good point. I'm not sure which driver is used in the test, but I'm guessing it's fanotify. I'll see if I can find a way to force one or the other and ensure we test both.

Reply:
try using an older ubuntu release, maybe ubuntu:focal